Product

```diff
@@ -0,0 +1,34 @@
+class Product: 
+    def __init__(self, sale_price, number_products, purchase_price, name, brand, image, barcode, instructions, 
+        size):
+        self.sale_price = sale_price # needs to be an integer
+        self.number_products = number_products # needs to be an integer
+        self.purchase_price = purchase_price # needs to be an integer
+        self.name = name # needs to be a string
+        self.brand = brand # needs to be a string 
+        self.image = image # needs to be a string
+        self.barcode = barcode # needs to be boolean 
+        self.instructions = instructions # needs to be a string 
+        self.size = size # needs to be an integer 
+
+
+    def design(): # we will need certain sizes, materials and colors
+        return ('We determine the size, texture and appropiate features the product should have to catch the attention of the users')
+
+        # we should return the appropiate combinations of the factors above 
+
+    def get_price_product(): # we will need to take into account factors like how difficult it was to 
+                            # create, how much it originally costs, and how much customers would be willing to 
+                            # pay for the product   
+        return ('We determine the adequate price of the product according to the cost of production and what customers would be comfortable with')
+        # we should return an adequate price, according to the factors above
+
+    def get_products_stock(): # we have to have access to the stock
```

Comment:
This shouldn't be a product method.

Reply:
I removed unnecessary methods to maintain a clean and understandable framework for my program.